### ISSUE TITLE
How to create a build/run configuration which creates a binary?

### ISSUE THREAD

**Issue by wizzard0** [asedunov/intellij-crystal-lang#50]
Noob question 2: how do I add the "build" configuration?
Just naively adding "build" to compiler parameters gets me

```
/opt/homebrew/Cellar/crystal/1.8.0/bin/crystal run --progress build /Users/user/dev/tmp/crtest/src/p3415crtest.cr
```
which predictably results in
```
Error: file 'build' does not exist
```

upd: for posterity
```
os: Monterey 13.3.1 (Apple Silicon)
crystal: 1.8.0
shards: 0.17.3
plugin: 0.7-2023.1
```

**Comment by asedunov** [comment]
Unfortunately that's not possible with at the moment. I'll add support for "Compile/Build" actions in the upcoming release

**Comment by asedunov** [comment]
Closed in favor of #52
